st2api/st2api/controllers/exp/aliasexecution.py

```diff
@@ -70,13 +71,21 @@ def _extract_parameters(self, action_alias_db, param_stream):
                                                             param_stream=param_stream)
         return parser.get_extracted_param_value()
 
-    def _schedule_execution(self, action_alias_db, params):
+    def _get_notify_field(self, payload):
```

Comment:
Having the user in payload may not work with executions. There we might tap into x-requester-user or some other x header to get this info. Just calling it out so we think about that.